docs/UserGuide.md

```diff
@@ -2,1012 +2,1166 @@
+- `copy 1` will copy the email of your first student in your student list to your clipboard.
+
+[Back to list of features](#features)
 
 ### Finding students by name or student ID: `find student`
 
-Finds students whose names or student IDs contain any of the given keywords(s).
+You can filter your student list to show only students whose names or student IDs contain any of the given keywords.
+
+**Command Format:**
 
-Format:
 ```
 find student <KEYWORD> [<MORE_KEYWORDS>]
 ```
+<div markdown="block" class="alert alert-info">:information_source: **Note:**<br>
 
-- You can find students using either `name` or `student ID` for the keyword(s).
-- Keywords are separated by spaces.
-- This command is key-insensitive.
+* You can find students using either `name` or `student ID` for the keyword(s).
+* Keywords are separated by spaces.
+* Keywords are case-insensitive.
+</div>
+<br>
 
 Examples:
+
 - `find student John` returns `John`.
 - `find student Alex Yu` returns `Alex Yeoh`, `Bernice Yu`.
 - `find student A123` returns students whose student ID contains `A123`, such as `A123` and `A1234567X`.
 
-[Back to list of commands](#commands)
+[Back to list of features](#features)
 
 ### Sorting list of students: `sort`
 
-Sorts list of students in student roster.
+You can sort your students either by their names or their student ID.
+
+**Command Format:**
 
-Format:
 ```
 sort <CATEGORY>
 ```
-
-- Current possible categories are `name` and `id`
+- `<CATEGORY>` is either `name` or `id`
 
 Examples:
-- `sort name` will sort list of students alphabetically according to name.
-- `sort id` will sort list of students alphanumerically according to student ID.
 
-![image](./images/UiStudentSortName.png)
-###### Figure 7
-![image](./images/UiStudentSortId.png)
-###### Figure 8
+- `sort name` will sort your list of students alphabetically according to name.
+- `sort id` will sort your list of students alphanumerically according to student ID.
+
+<img src="./images/UiStudentSortName.png" width="650" height="529"/>
+<br>
+<img src="./images/UiStudentSortId.png" width="650" height="529"/>
+<br>
+
+[Back to list of features](#features)
 
-[Back to list of commands](#commands)
+### Display a student's information: `select`
 
-### Displays a student's information: `select`
+This command allows you to select a student and display their full particulars.
+The selected student will be shown like this:
 
-![image](./images/UiStudentView.png)
-###### Figure 9
+<img src="./images/UiStudentView.png" width="650" height="529"/>
 
-Select a particular student and display his / her particulars.
+**Command Format:**
 
-Format:
 ```
 select <INDEX>
 ```
 
 Examples:
-- `select 1` will select the 1st student in the student list and display his / her particulars on the view panel on the right.
 
-[Back to list of commands](#commands)
+- `select 1` will select the first student in the student list and display his / her particulars on the view panel on the right.
```

Comment:
removed mention of "view panel", to just be  "select the first student in the student list and display his / her particulars." .  

good catch. i actually meant to remove all mentions of right pane etc. since its abit extra as the user just needs to know that they will see it. help to see if theres any more of these.